### ISSUE TITLE
Merging in RQDA code/files

### ISSUE THREAD

**Issue by The-Pezman** [ropenscilabs/qcoder#226]
As I imagine many on this project know, RQDA is another R package for qualitative data analysis, but it hasn't been available on CRAN for some time now. There has not been any significant development on the package for some time ([despite my urging](https://github.com/Ronggui/RQDA/issues/38)), and since learning about qcoder I'm beginning to think that a brand-new package, built with a Shiny interface, might be the way to go.

It would be great if qcoder could import RQDA files, so that it can effectively serve as RQDA's successor and make the switch easier for RQDA users. That way, the old package can be completely retired, and qcoder will be able to take on RQDA's fans and users.

What is needed to help make this happen?

**Comment by elinw** [comment]
Hi, that's a very nat idea, interoperability is on our road map.  Our motivation document actually discusses RQDA.   What a crazy thread. 
I assume the data are in a data base, right? 
What would be great is if you could upload some example files we could look at them and see how they correspond to the simpler structure we are using. Currently I am working on the project of switching the storage of data to git (see the ROpenSci package for doing that).  Right now we store the data in RDS files (but are shifting away from that to flat files in git) and have UIs to export data to csv files.  We read data in from csv (for codes, units, links ) and txt (for documents though other formats should work). 
I also wrote some code to import the data from a spreadsheet of data from the reddit API.  
Basically if we have the files it should be relatively straightforward to write functions to import them. 
Feel free to give it a try, just keep in mind that we will want to be writing to flat csv files.

**Comment by elinw** [comment]
https://github.com/ropenscilabs/qcoder/issues/193

So one of the big things that would be nice to have and related to this is to change the qcoder structures so that they recode who did the coding. But that is one of the purposes of moving to git since it will record that in the history.
